pages/linux/xbps.md

```diff
@@ -0,0 +1,23 @@
+# xbps
+
+> The X Binary Package System (or xbps) is the binary package system used by Void Linux.
+
+- Install packages:
+
+`xbps-install -S package_name1 package_name2`
```

Comment:
Could `package_name1` and `package_name2` be each surrounded in `{{` and `}}` please? This special syntax indicates to a tldr-client that it's something that may be edited.

The same goes for `package_name` in the examples below.

Reply:
fixed